src/component/AttributionComponent.ts

```diff
@@ -61,12 +61,8 @@ export class AttributionComponent extends Component<IComponentConfiguration> {
         const imageBy: string = compact ? `${username}` : `image by ${username}`;
         const imageByContent: vd.VNode = vd.h("div.AttributionUsername", { textContent: imageBy }, []);
 
-        const date: string[] = new Date(capturedAt).toDateString().split(" ");
-        const formatted: string = (date.length > 3 ?
-            compact ?
-                [date[3]] :
-                [date[1], date[2] + ",", date[3]] :
-            date).join(" ");
```

Comment:
Btw, the current implementation looks a bit overly complex to me or has dead code. Since this will always be true:
```TypeScript
new Date(capturedAt).toDateString().split(" ").length > 3
```
Unless, `Date.toDateString()` is JavaScript runtime implementation specific, and thus behaves differently on different browsers. But, I am not aware of that.